src/tests/unit/unit.test.ts

```diff
@@ -0,0 +1,43 @@
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
```

Comment:
I don't think you need to import `describe` and `it` once you run mocha you get them as a global function, if it is because you had a listing problem you need to specify in eslintrc that you use mocha

Reply:
right, done